apps/openmw/mwgui/exposedwindow.cpp

```diff
@@ -2,12 +2,12 @@
 
 namespace MWGui
 {
-    MyGUI::VectorWidgetPtr ExposedWindow::getSkinWidgetsByName (const std::string &name)
+    MyGUI::VectorWidgetPtr Window::getSkinWidgetsByName (const std::string &name)
```

Comment:
Why all the renames?

Reply:
The point is to replace implementation for the basic Window class. Otherwise I need turn everything for ExposedWindows - a lot of mess with layout files.

Probably it is possible implement a custom button class instead of using custom layout properties, but I am not sure that is is better.